src/bitcoin-cli.cpp

```diff
@@ -273,8 +273,10 @@ class GetinfoRequestHandler: public BaseRequestHandler
         result.pushKV("timeoffset", batch[ID_NETWORKINFO]["result"]["timeoffset"]);
 
         UniValue connections(UniValue::VOBJ);
-        connections.pushKV("in", batch[ID_NETWORKINFO]["result"]["connections_in"]);
-        connections.pushKV("out", batch[ID_NETWORKINFO]["result"]["connections_out"]);
+        const UniValue in{batch[ID_NETWORKINFO]["result"]["connections_in"]};
+        const UniValue out{batch[ID_NETWORKINFO]["result"]["connections_out"]};
+        if (!in.isNull()) connections.pushKV("in", in);
```

Comment:
If we do this, I'd prefer to add a comment describing why these `isNull` checks are done for these fields and not for, say `connections` itself. It won't be obvious in the future.

Reply:
Good idea. Added.